### PR TITLE
feature: Tree accordion (#2387)

### PR DESCRIPTION
**Pull request by baiyaaaaa**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

每次只保持一个同级父节点的展开（feature #2387）

### CODE REVIEW

File: examples/docs/en-US/tree.md [hunk 1]
```diff
@@ -214,6 +244,69 @@ Used for node selection. In the following example, data for each layer is acquir
 ```
 :::
 
+### Accordion
+
+Only one equative node can be expanded in the meantime.
```
Comment (Leopoldthecoder) [inline examples/docs/en-US/tree.md]: Only one node among the same level can be expanded at one time.

File: examples/docs/zh-CN/tree.md [hunk 2]
```diff
@@ -245,6 +338,7 @@
 | check-strictly | 在显示复选框的情况下，是否严格的遵循父子不互相关联的做法，默认为 false | boolean | — | false |
 | default-checked-keys | 默认勾选的节点的 key 的数组 | array | — | — |
 | filter-node-method | 对树节点进行筛选时执行的方法，返回 true 表示这个节点可以显示，返回 false 则表示这个节点会被隐藏 | Function(value, data, node) | — | — |
+| accordion | whether one equative node can be expanded in the meantime | boolean | — | false |
```
Comment (Leopoldthecoder) [inline examples/docs/zh-CN/tree.md]: 这个不是中文文档吗= =

**Comment by baiyaaaaa** [comment]
done

**Comment by xianxingg** [comment]
`每次只保持一个同级父节点的展开`可能导致展开的节点不是目前选中节点的子节点。比如

+ A
    - A1
    - A2
+ B
    - B1
    - B2
+ C

对于此结构，假设最开始所有的子选项均未展开。
点击A节点，将展开A1和A2。
点击B节点，将展开B1和B2，同时A1和A2会折叠。
点击C节点，**B1和B2不会折叠**。

这会不会产生歧义？

**Pull request merged**